main.go

```diff
@@ -138,7 +138,10 @@ func main() {
 			orgs.GET("/:orgid/allowed/secrets/:type", api.ListAllowedSecretTypes)
 		}
 		//v1.GET("/clusters/gke/:projectid/:zone/serverconf", cluster.GetGkeServerConfig) // todo think about it and move
-		v1.GET("/token", auth.GenerateToken)
+		v1.GET("/token", auth.GenerateToken) // Should be POST /tokens
```

Comment:
Maybe be more specific that this is generating a token (like `/generateToken`) or change to POST

Reply:
Once the UI can handle token generation I will change it to POST.